### ISSUE TITLE
Does the return value of twinspan() include the classification of species

### ISSUE THREAD

**Issue by matutosi** [zdealveindy/twinspanR#5]
I would like to get the classification of species as a return value of the function twinspan(), but I can not find it.
If twinspan() returns it, please let me know the way to get the classification of species.

**Comment by zdealveindy** [comment]
No, I am afraid it doesn't return the classification of species, only classification of samples. If I understand it right, you need to identify indicator species for individual divisions, right? If the original "twinspan.exe" algorithm is returning them, I may be able to print them out, but I haven't seen it there. 
Please also check "twinspan" library of Jari Oksanen (https://github.com/jarioksa/twinspan) to see if you can find it there.
David
